xcode/Subconscious/Shared/Models/StoryCombo.swift

```diff
@@ -28,7 +28,7 @@ struct StoryCombo: Hashable, Identifiable, Codable, CustomStringConvertible {
 extension MemoEntry {
     init?(_ story: StoryCombo) {
         // Order by slug alpha
-        let (x, y) = Func.block({
+        let (x, y) = Func.run({
```

Comment:
Interested to know the thinking on this approach vs an IIFE, I know it's divisive it is possible to do:

```swift
{
    if story.entryA.address < story.entryB.address {
        return (story.entryA, story.entryB)
    } else {
        return (story.entryB, story.entryA)
    }
}()
```

Reply:
I prefer having this small utility function, because it announces what it is doing up-front.

You can't see that something is an IFFE until you get to the end of the block (which can potentially be long). I find announcing intent up-front to be more readable, and analogous to `do` blocks, etc. It's a personal quirk, but not unreasonable, I think!

Side-note: we could go a step further and make `run` a top-level function, making it very similar to a `do` that is an expression:

```swift
run {
   // Some code
}
```

But I decided to keep it all under the `Func` namespace for now.

Reply:
That makes sense! I don't have a strong view, so just curious to hear yours. the top-level signature for `run` is tempting... but not exactly a pressing concern.